public/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Mock Real Estate Platform</title>
+  <link rel="stylesheet" href="css/house.css" />
```

Comment:
This needs to be an absolute path from your root: `href="/css/house.css"`

The spa-server-route module is serving up index.html for any routes your express router doesn't explicitly support. But this means that if it's serving it up from something like `/houses/somestreetname`, the relative paths inside your index.html file will be resolved to `/houses/css/house.css` or whatever else the resource name is, and it won't be found by the express.static middleware in the public directory like it expects. Make any resource paths like this hard-coded as if they're from the root (I believe you have some image files with similar paths) and it should fix the way your pages look if you load the server from your `/houses/somestreetname` routes.